app/controllers/ChannelTestsController.scala

```diff
@@ -162,6 +163,7 @@ abstract class ChannelTestsController[T <: ChannelTest[T] : Decoder : Encoder](
       logger.info(s"${request.user.email} is updating $channel/'${test.name}'")
       dynamoTests
         .updateTest(test, channel, request.user.email)
+        .flatMap(_ => dynamoTestsAudit.createAudit(test, request.user.email))
```

Comment:
This means after the update has been written to the main table, we then write to the audit table.
`flatMap` ([from zio](https://zio.dev/overview/basic-operations/#chaining)) chains these operations together in sequence